op-mode-definitions/containers.xml.in

```diff
@@ -40,19 +40,13 @@
         <properties>
           <help>Show containers</help>
         </properties>
-        <command>sudo ${vyos_op_scripts_dir}/containers_op.sh --show-containers</command>
+        <command>sudo ${vyos_op_scripts_dir}/containers_op.py --all</command>
         <children>
           <leafNode name="image">
             <properties>
-              <help>Show available container images</help>
-            </properties>
-            <command>sudo ${vyos_op_scripts_dir}/containers_op.sh --show-images</command>
-          </leafNode>
-          <leafNode name="network">
-            <properties>
-              <help>Show available container networks</help>
+              <help>Delete container image</help>
             </properties>
-            <command>sudo ${vyos_op_scripts_dir}/containers_op.sh --show-networks</command>
```

Comment:
I guess this is a typo and should be networks.

Reply:
For some reason, leafNode "network" was deleted.
Fixed.